mail/protocols/pop3/src/test/java/com/fsck/k9/mail/store/pop3/Pop3FolderTest.kt

```diff
@@ -225,15 +225,15 @@ class Pop3FolderTest {
             MIME-Version: 1.0
             Content-type: text/plain
             Content-Transfer-Encoding: 7bit
-            
+
             this is some test text.
             """.trimIndent().crlf().byteInputStream()
         folder.open()
         val messageList = setupMessageFromServer()
         val fetchProfile = FetchProfile()
         fetchProfile.add(FetchProfile.Item.BODY)
         stubbing(connection) {
-            on { readLine() } doReturn("1 100") doReturn(".")
+            on { readLine() } doReturn ("1 100") doReturn (".")
```

Comment:
It's unfortunate that ktlint doesn't catch this. We probably want to change this to

```kotlin
on { readLine() } doReturn "1 100" doReturn "."
```